oceanwaters/doc/setup_dev_env.md

```diff
@@ -65,7 +65,7 @@ a stable version of PLEXIL compatible with OceanWATERS and suitable for general
 use.
```

Comment:
Wouldn't it be better to explicitly state the `releases/plexil-4` branch during checkout. Just in case, the default branch was modified by PLEXIL team in the future? `git clone --branch releases/plexil-4 https://git.code.sf.net/p/plexil/git plexil`

Reply:
It was designed to stay this way, but this is safer, so I put it in.